### DISCUSSION TITLE
See no elements on a blank window

### DISCUSSION THREAD

**Discussion by squadgazzz** [TomSchimansky/CustomTkinter#1933] [Q&A]
Running a simple example
```
import customtkinter

customtkinter.set_appearance_mode("System")  # Modes: system (default), light, dark
customtkinter.set_default_color_theme("blue")  # Themes: blue (default), dark-blue, green

app = customtkinter.CTk()  # create CTk window like you do with the Tk window
app.geometry("400x240")

def button_function():
    print("button pressed")

# Use CTkButton instead of tkinter Button
button = customtkinter.CTkButton(master=app, text="CTkButton", command=button_function)
button.place(relx=0.5, rely=0.5, anchor=customtkinter.CENTER)

app.mainloop()
```
As a result, see only a blank window with no elements.
<img width="403" alt="image" src="https://github.com/TomSchimansky/CustomTkinter/assets/22964585/d4c0fb5d-a9c3-4405-90e4-cb8791cf2689">
Does anyone know how to fix it? MacOS Ventura, Intel. Python 3.9.6. I've already tried other examples with the same output.
v5.1.3

Also tried with v5.2.0 on a different Intel machine: the same result.

`python -m tkinter` resulted into something weird:
<img width="154" alt="image" src="https://github.com/TomSchimansky/CustomTkinter/assets/22964585/69036eeb-173b-44ae-85c1-69b6f0993e0c">

Then I tried to install tcl-tk via brew and to link it
```
brew install tcl-tk
brew link tcl-tk --force
```
And that also didn't help...

**Comment by Akascape** [comment]
@squadgazzz Duplicate of https://github.com/TomSchimansky/CustomTkinter/issues/1690 and https://github.com/TomSchimansky/CustomTkinter/issues/1915

This is a big problem with tkinter and macosx, basically due to older python versions, solution is to upgrade python or create your venv with python 3.11+
